### ISSUE TITLE
Press enter to type in values not working in step and harmonic editors

### ISSUE THREAD

**Issue by ironcross32** [surge-synthesizer/surge#7211]
This information is found on the About screen, which you get to from the bottom right menu.
Surge XT 1.3.nightly.174ba3f
* Version:
* Plugin Type: [VST2, VST3, AU]
* VST3
* Bitness: [32-bit or 64-bit]
64-bit

1. Go to either the step sequencer or the additive waveform editor
2. Hit enter on either a step value or one of the harmonic controls
3. Note that it doesn't open the typein

**Comment by baconpaul** [comment]
Oh thanks that should be an easy enough fix. I'll tag it into the 1.3 milestone.

**Comment by baconpaul** [comment]
This will be fixed in the next nightly later today.
